include/libcloudph++/blk_2m/terminal_vel_formulae.hpp

```diff
@@ -160,20 +160,19 @@ namespace libcloudphxx
           * si::cubic_metres / si::kilograms // to make it dimensionless
           * real_t(1000) // mass of the drop in grams
           * (
-	    alpha_fall(d1<real_t>() / real_t(2))
-            * std::pow(c_md<real_t>() * si::cubic_metres / si::kilograms * real_t(1000), beta_fall(d1<real_t>() / real_t(2)))
+	    alpha_fall(d1<real_t>() / real_t(4)) // average radius between 0 and d1
```

Comment:
These lines have two bugs in them 1) there were missing parentheses so the /2 was only applied to the second diameter, not the first, and 2) it should actually be /4 because the `alpha_fall` and `beta_fall` functions expect a droplet radius not droplet diameter.